builtin/providers/google/resource_compute_instance_group.go

```diff
@@ -0,0 +1,301 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceComputeInstanceGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeInstanceGroupCreate,
+		Read:   resourceComputeInstanceGroupRead,
+		Update: resourceComputeInstanceGroupUpdate,
+		Delete: resourceComputeInstanceGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"fingerprint": &schema.Schema{
```

Comment:
Yes that's exactly right.

On Wed, Dec 2, 2015 at 4:06 AM, Alex Crowe <notifications@github.com> wrote:

> In builtin/providers/google/resource_compute_instance_group.go
> <https://github.com/hashicorp/terraform/pull/4087#discussion_r46390373>:
>
> > +		Delete: resourceComputeInstanceGroupDelete,
> > +
> > +		Schema: map[string]*schema.Schema{
> > +			"name": &schema.Schema{
> > +				Type:     schema.TypeString,
> > +				Required: true,
> > +				ForceNew: true,
> > +			},
> > +
> > +			"description": &schema.Schema{
> > +				Type:     schema.TypeString,
> > +				Optional: true,
> > +				ForceNew: true,
> > +			},
> > +
> > +			"fingerprint": &schema.Schema{
>
> Ok I understand, so you would want the manually added D to persist and
> Terraform to only manage the instances that have been declared?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hashicorp/terraform/pull/4087/files#r46390373>.
>